save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/cosv/RawCosvFile.kt

```diff
@@ -34,6 +37,8 @@ class RawCosvFile(
     var organization: Organization,
     @Enumerated(EnumType.STRING)
     var status: RawCosvFileStatus,
+    @Formula("status = 'UPLOADED' AND LOWER(file_name) LIKE '%_.zip'")
```

Comment:
I don't think that need to check additonally status -- file name will be enough

Reply:
We work with archives when they have only `UPLOADED` status. They can't have any other status. So to show this I decided that it would be better to also add status check. Should I remove this check? Or I can rename this field to `isUploadedZip`?

Reply:
I think better to remove: it also remove hardcoding on status name  (`UPLOADED`)

Reply:
Done